### PR TITLE
Change hears to responds. Brehm is super annoying

### PR DESCRIPTION
**Pull request by luke-eisenbraun**

### CODE REVIEW

File: scripts/youtried.coffee [hunk 1]
```diff
@@ -6,5 +6,5 @@
 #   Mike Jensen
 
 module.exports = (robot) ->
```
Comment (jcox92) [inline scripts/youtried.coffee]: :+1:

File: scripts/umad.coffee [hunk 2]
```diff
@@ -6,5 +6,5 @@
 
 module.exports = (robot) ->
   robot.hear /(you|u) mad\b/i, (msg) ->
```
Comment (jcox92) [inline scripts/umad.coffee]: :+1:

File: scripts/snakeeyes.coffee [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # snakeeyes is back ... and this time it's personal!
 
 module.exports = (robot) ->
-  robot.hear /snake ?eyes/i, (msg) ->
```
Comment (jcox92) [inline scripts/snakeeyes.coffee]: :-1:

File: scripts/sloth-me.coffee [hunk 4]
```diff
@@ -60,5 +60,5 @@ sloths = [
 ]
 
 module.exports = (robot) ->
-  robot.hear /sloth me/i, (msg) ->
+  robot.respond /sloth me/i, (msg) ->
```
Comment (jcox92) [inline scripts/sloth-me.coffee]: :+1:

File: scripts/likeaboss.coffee [hunk 5]
```diff
@@ -10,5 +10,5 @@ images = [
   ]
 
 module.exports = (robot) ->
-  robot.hear /like a boss/i, (msg) ->
+  robot.respond /like a boss/i, (msg) ->
```
Comment (jcox92) [inline scripts/likeaboss.coffee]: :+1:

File: scripts/hyphen.coffee [hunk 6]
```diff
@@ -2,7 +2,7 @@
 #   Keaton Greve (keaton.greve)
 
 module.exports = (robot) ->
-  robot.hear /([^\s]*)[-|\s]+ass\s([^\s]*)/i, (msg) ->
+  robot.respond /([^\s]*)[-|\s]+ass\s([^\s]*)/i, (msg) ->
```
Comment (jcox92) [inline scripts/hyphen.coffee]: :-1:

File: scripts/friday.coffee [hunk 7]
```diff
@@ -3,6 +3,6 @@
 # Friday - Rebecca Black would be proud
 
 module.exports = (robot) ->
-  robot.hear /.*friday.*/i, (msg) ->
+  robot.respond /.*friday.*/i, (msg) ->
```
Comment (jcox92) [inline scripts/friday.coffee]: :+1:

File: scripts/food.coffee [hunk 8]
```diff
@@ -27,7 +27,7 @@ module.exports = (robot) ->
     nowHours = now.getHours()    
     msg.send msg.random gifs  if nowHours is 11 or nowHours is 12
 
-  robot.hear /lunch/i, (msg) ->
+  robot.respond /lunch/i, (msg) ->
```
Comment (jcox92) [inline scripts/food.coffee]: :+1:

File: scripts/business.coffee [hunk 9]
```diff
@@ -2,5 +2,5 @@
 # 	Shows a haha, business image when anybody says business.
 
 module.exports = (robot) ->
-  robot.hear /business/i, (msg) ->
```
Comment (jcox92) [inline scripts/business.coffee]: :+1:

File: scripts/boomcartdone.coffee [hunk 10]
```diff
@@ -5,5 +5,5 @@
 # http://i.imgur.com/SMuFDx4.gif
 
 module.exports = (robot) ->
-  robot.hear /.*(boom).*/i, (msg) ->
```
Comment (jcox92) [inline scripts/boomcartdone.coffee]: :+1:

File: scripts/alot.coffee [hunk 11]
```diff
@@ -25,5 +25,5 @@ alot = [
   "https://s3.amazonaws.com/hudl-internal-assets/alot/i__m_sad__alot_by_celestialbeast-d3f0uv1.jpg"]
 
 module.exports = (robot) ->
-  robot.hear /alot/i, (msg) ->
```
Comment (jcox92) [inline scripts/alot.coffee]: :-1:

**Comment by bclymer** [comment]
We're reaching an agreement. Closing this PR.

**Pull request closed without merging**